src/template.html

```diff
@@ -190,52 +195,70 @@
-                        uprzyjemnić naukę niemieckiego? Naszym zdaniem jest to
-                        dzielenie jej z kimś bliskim. Zapisz się na zajęcia z
-                        koleżanką z pracy, bratem, żoną, przyjacielem, mamą.
-                        Gwarantujemy, że przyjemność z nauki i motywacja do niej
-                        jeszcze bardziej wzrosną, a cena zmaleje o 50%!
-                    </p>
-                </div>
-                <div class="offer-list">
-                    <ul>
-                        <p>
-                            Bezpłatne 45-minutowe zajęcia próbne, na których omówimy
-                            Twoje potrzeby i plan działania
-                        </p>
-                        <p>
-                            Godziny zajęć dostosowane do Twoich potrzeb, umawiane
-                            indywidualnie z lektorem
-                        </p>
-                        <p>
-                            Dostęp do folderu z Twoim słowniczkiem, podręcznikiem i
-                            wieloma innymi materiałami, z których możesz korzystać na
-                            zajęciach i w domu
-                        </p>
-                        <p>Możliwość zajęć w parze za tę samą cenę</p>
-                        <p>Przede wszystkim przyjemna i efektywna nauka języka! ☺</p>
-                    </ul>
+                    <h3 class="offer-subtitle">
+                        W ramach naszej oferty mozesz wybrać:
+                    </h3>
+
+                    <div class="offer-list">
+                        <ul>
+                            <p>
+                                Zajęcia indywidualne - największa elastyczność i
+                                dopasowanie zagadnień omawianych na zajęciach tylko i
+                                wyłącznie do Twoich potrzeb!
+                            </p>
+
+                            <p>
+                                Zajęcia grupowe - kameralne 3-osobowe grupy oraz
+                                najniższa cena!
+                            </p>
+                            <p>
+                                A może zajęcia w parach? Wspólna nauka z koleżanką z
+                                pracy, bratem, żoną, przyjacielem, mamą to nie tylko
+                                przyjemność, ale też okazja żeby zaoszczędzić!
+                            </p>
+                        </ul>
+                    </div>
+
+                    <img src="images/cennik.png" alt="germani w liczbach" class="headers" />
 
-                   <img src="images/price.svg" alt="cena" class="price">
+                    <h3 class="offer-subtitle">Pliki do pobrania:</h3>
+
+                    <div class="download-buttons">
+                        <a class="btn" href="./Regulamin GermAni.pdf" target="_blank">Regulamin</a>
+                        <a class="btn" href="./Formularz zgłoszeniowy GermAni.docx" target="_blank">
```

Comment:
I think href cannot have blank spaces in the path

Reply:
works fine - no changes